src/app/public/testing/split-view/split-view-testing.module.ts

```diff
@@ -6,9 +6,16 @@ import {
   SkySplitViewModule
 } from '@skyux/split-view';
 
+import {
+  SkyThemeService
+} from '@skyux/theme';
+
 @NgModule({
   exports: [
     SkySplitViewModule
+  ],
+  providers: [
+    SkyThemeService
```

Comment:
☝️ Not a part of this PR, but the prior `@skyux/split-view` release started depending on `SkyThemeService`, but this fixture was not updated.